cmd/serve/main.go

```diff
@@ -127,6 +128,7 @@ func (cmd *servecmd) Run() {
 		logrus.Infof("starting wireguard server on :%d", conf.WireGuard.Port)
 
 		wgconfig := &wgembed.ConfigFile{
+			// TODO: add MTU config
```

Comment:
This can be done at a later time. What is more important is the client configuration generated by the admin-panel.

Reply:
I would rather have the server configuration in working order as well.
You would have to make a change to the https://github.com/freifunkMUC/wg-embed project first.


If this is not possible, maybe remove the server part for now, and just start with the `clientConfig.mtu` part.